src/index.js

```diff
@@ -135,6 +135,25 @@ exports.handler = constructHandler(async webhook => {
     addRecheckComment
   } = applyToken(token);
 
+  logger.info("Obtaining the list of commits for the pull request");
```

Comment:
This code was moved from further up the file as it is needed to obtain the SHA for the most recent commit in order to report errors relating to malformed JSON via the status API